app/javascript/src/QuickStarts/utils/progressTracker.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+/**
+ * This is a collection of wrappers around localStorage to ease getting information about Quickstarts state
+ */
+
+function getActiveQuickstart (): null | string {
```

Comment:
I think you can do something like:

```
function getActiveQuickstart(): null | string {
  const quickstartId = localStorage.getItem('quickstartId')
  return quickstartId || null
}
```

From MDN (https://developer.mozilla.org/en-US/docs/Web/API/Storage/getItem), it returns: `A string containing the value of the key. If the key does not exist, null is returned.`

I don't think there's a need to `JSON.parse` as it looks like it's an ID you are saving and then retrieving.

And no need to check for length as the `getItem` call will already return either the string or null if the key does not exist.  The reason for the short circuit logic in the `return` is to catch the edge case of `quickstartId` being an empty string.